samplers/parser.go

```diff
@@ -221,7 +221,7 @@ func ParseEvent(packet []byte) (*UDPEvent, error) {
 	if len(textChunk) != textExpectedLength {
 		return nil, errors.New("Invalid event packet, actual text length did not match encoded length")
 	}
-	ret.Text = string(textChunk)
+	ret.Text = strings.Replace(string(textChunk), "\\n", "\n", -1)
```

Comment:
oh - yeah, it's certainly possible. event messages can contain some markdown constructs, so having weird richly formatted text is not out of the question